core/src/main/java/tc/oc/pgm/api/map/MapInfo.java

```diff
@@ -109,12 +109,19 @@ default String getStyledNameLegacy(MapNameStyle style, @Nullable CommandSender s
   Collection<MapTag> getTags();
 
   /**
-   * Get a {@link Component} that represents this map's gamemode name.
+   * Get a {@link Component} that represents this map's custom game title.
    *
    * @return A component of the gamemode name if defined or null.
    */
   Component getGamemode();
 
+  /**
+   * Get a {@link Collection<Gamemode>} that represents this map's gamemodes.
+   *
+   * @return A component of gamemodes if defined or null.
```

Comment:
```suggestion
   * @return The list of defined gamemodes, empty list if none are defined.
```

Don't lie in the javadocs, makes the APIs hard to use

Reply:
Doesn't look resolved